### PR TITLE
Adding new committer checklist

### PR DESCRIPTION
**Pull request by fkautz**

### CODE REVIEW

File: docs/new-committer-checklist.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# New Committer Checklist
+
+- [ ] Announcement in General Meeting
+- [ ] Change in OWNERS.md
+- [ ] Access to Packet.net
+- [ ] Access to GCE
+- [ ] Access to Azure
+- [ ] Access to FOSSA
+- [ ] Access to Netlify
```
Comment (nickolaev) [inline docs/new-committer-checklist.md]: https://github.com/networkservicemesh/networkservicemesh
https://github.com/networkservicemesh/site
https://github.com/orgs/networkservicemesh/people

Comment (edwarnicke) [reply docs/new-committer-checklist.md]: [ ] Access to twitter account
[ ] Access to docker hub account
[ ] Add as Admin to github org
[ ] Add as Admin to github repos

Comment (fkautz) [reply docs/new-committer-checklist.md]: I've added these, can you verify the changes are sufficient?

**Comment by fkautz** [comment]
@edwarnicke @nickolaev can your review it again?

File: docs/new-committer-checklist.md (hunk 1 above)
Comment (edwarnicke) [reply docs/new-committer-checklist.md]: 👍

**Pull request merged**